bench-tps/Cargo.toml

```diff
@@ -39,7 +39,7 @@ solana-rpc-client-nonce-utils = { workspace = true }
 solana-runtime = { workspace = true, features = ["dev-context-only-utils"] }
 solana-sdk = { workspace = true }
 solana-streamer = { workspace = true }
-solana-tps-client = { workspace = true }
+solana-tps-client = { workspace = true, features = ["bank-client"] }
```

Comment:
Do we need a bank client at all in bench-tps? It should be relevant only for integration tests as far as I see (maybe I'm wrong)

Reply:
correct, I see it's only needed in dev-dependencies